.github/workflows/sass.yaml

```diff
@@ -45,6 +45,14 @@ jobs:
         run: |
           ! grep "\$govuk-" .tmp/all.css
 
+      - name: Save compiled Sass
+        uses: actions/upload-artifact@v4.3.1
+        if: ${{ !cancelled() }}
```

Comment:
**nitpick** Thinking we could add a `continue-on-error` to avoid that step failing the job if the upload has an issue. This `continue-on-error` would have to be conditionned to the previous step being a success, though. Otherwise, I think it'd make test failures not fail the job.

Reply:
Why do we need to guard against the job being cancelled?

Reply:
Following GitHub's recommendations to prefer `!cancelled()` to `always()` to ensure a step runs regardless of the previous steps. Makes sense to me as if the job was cancelled, we don't care about uploading artifacts anyways. Happy to drop a comment on the first one to remind of [GitHub's recommendations](https://docs.github.com/en/actions/learn-github-actions/expressions#always).